lib/tasks_private/spec.rake

```diff
@@ -18,8 +18,31 @@ namespace :spec do
   desc "Prepare all specs"
   task :setup => [:initialize, :setup_db]
 
+  desc <<~DESC
+    Generate table list for schema specs
+
+    Generates a list of tables from the `spec/dummy/db/schema.rb` to be used in
+    the `schema_dumper_spec.rb`.
+  DESC
+  task :table_list => "spec:migrations:complete_up" do
```

Comment:
Is this task meant to be a "public" task?  Adding a `desc` adds it to rake -T making it public.  If it's expected to be private, prefer a regular comment over `desc`

Reply:
@Fryguy yeah, it is.  The intent is that it is used to re-generate the `table_list.txt` for when a new migration adds a table.  It is designed so that a developer doesn't have to add it to the file manually, and just re-runs that task.

That said, I was going to add a comment into the spec (which I forgot to do), that explained that if this spec fails, please run this task to re-generate the file and try again.

Reply:
> That said, I was going to add a comment into the spec (which I forgot to do), that explained that if this spec fails, please run this task to re-generate the file and try again.

Perfect.  That was going to be my next suggestion.

Reply:
I went with a slightly different solution, by adding the comment as the error message.  Figured it allowed the action items o be seen in travis as well.

Might want to consider some updates to the `README.md` to inform developers in other ways that this will be a new pattern that is needed, but that probably can happen in a followup PR.